sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -0,0 +1,800 @@
+};
+
+// @public
+export type AnalyzeActionParameters<ActionName extends AnalyzeActionName> = {
+    EntityLinking: EntityLinkingAction;
+    EntityRecognition: EntityRecognitionAction;
+    PiiEntityRecognition: PiiEntityRecognitionAction;
+    KeyPhraseExtraction: KeyPhraseExtractionAction;
+    SentimentAnalysis: SentimentAnalysisAction;
+    LanguageDetection: LanguageDetectionAction;
+}[ActionName];
+
+// @public
+export type AnalyzeResult<ActionName extends AnalyzeActionName> = {
+    EntityLinking: EntityLinkingResult[];
+    EntityRecognition: EntityRecognitionResult[];
+    PiiEntityRecognition: PiiEntityRecognitionResult[];
+    KeyPhraseExtraction: KeyPhraseExtractionResult[];
+    SentimentAnalysis: SentimentAnalysisResult[];
+    LanguageDetection: LanguageDetectionResult[];
+}[ActionName];
+
+// @public
+export interface AssessmentSentiment {
+    confidenceScores: TargetConfidenceScores;
+    isNegated: boolean;
+    length: number;
+    offset: number;
+    sentiment: TokenSentimentLabel;
+    text: string;
+}
+
+export { AzureKeyCredential }
+
+// @public
+export interface DetectedLanguage {
+    confidenceScore: number;
+    iso6391Name: string;
+    name: string;
+}
+
+// @public
+export type DocumentSentimentLabel = "positive" | "neutral" | "negative" | "mixed";
+
+// @public
+export interface DocumentWarning {
+    code: WarningCode;
+    message: string;
+}
+
+// @public
+export interface Entity {
+    category: string;
+    confidenceScore: number;
+    length: number;
+    offset: number;
+    subCategory?: string;
+    text: string;
+}
+
+// @public
+export type EntityLinkingAction = ActionPrebuilt & {
+    stringIndexType?: StringIndexType;
+};
+
+// @public
+export type EntityLinkingErrorResult = TextAnalysisErrorResult;
+
+// @public
+export type EntityLinkingResult = EntityLinkingSuccessResult | EntityLinkingErrorResult;
+
+// @public
+export interface EntityLinkingSuccessResult extends TextAnalysisSuccessResult {
+    readonly entities: LinkedEntity[];
+}
+
+// @public
+export type EntityRecognitionAction = ActionPrebuilt & {
+    stringIndexType?: StringIndexType;
+};
+
+// @public
+export type EntityRecognitionErrorResult = TextAnalysisErrorResult;
+
+// @public
+export type EntityRecognitionResult = EntityRecognitionSuccessResult | EntityRecognitionErrorResult;
+
+// @public
+export interface EntityRecognitionSuccessResult extends TextAnalysisSuccessResult {
+    readonly entities: Entity[];
+}
+
+// @public
+export type KeyPhraseExtractionAction = ActionPrebuilt & {};
+
+// @public
+export type KeyPhraseExtractionErrorResult = TextAnalysisErrorResult;
+
+// @public
+export type KeyPhraseExtractionResult = KeyPhraseExtractionSuccessResult | KeyPhraseExtractionErrorResult;
+
+// @public
+export interface KeyPhraseExtractionSuccessResult extends TextAnalysisSuccessResult {
+    readonly keyPhrases: string[];
+}
+
+// @public
+export enum KnownErrorCode {
+    // (undocumented)
```

Comment:
👀

Reply:
I opened https://github.com/Azure/autorest.typescript/issues/1368 a while back to track this issue. /cc @JonathanCrd